pipelines/folioRancher/folioTestingTools/folioSchemaCompare.groovy

```diff
@@ -0,0 +1,221 @@
+#!groovy
+import hudson.util.Secret
+import org.jenkinsci.plugins.workflow.libs.Library
+import java.time.*
+import org.folio.rest.GitHubUtility
+import org.folio.Constants
+import groovy.json.JsonSlurperClassic
+
+@Library('pipelines-shared-library@RANCHER-838') _
+
+String getOkapiVersion(folio_repository, folio_branch) {
```

Comment:
Better to move the function to the bottom of the script or even to the separate lib folioCommon for example.